Sources/TransferMethod/AddTransferMethodPresenter.swift

```diff
@@ -113,6 +113,13 @@ final class AddTransferMethodPresenter {
                                                                          transferMethodCurrency: currency)
                 .build()
 
+        case "WIRE_ACCOUNT":
```

Comment:
We should add another case here, instead, add this case to BANK_ACCOUNT. And in core sdk, add another signature in the builder, something like:
```
 case "BANK_ACCOUNT", "WIRE_ACCOUNT":
            hyperwalletTransferMethod = HyperwalletBankAccount.Builder(transferMethodCountry: country,
                                                                       transferMethodCurrency: currency,
                                                                       transferMethodProfileType: profileType,
                                                                       transferMethodType: transferMethodTypeCode)
)
```

Reply:
Also in ListTransferMehodPresenter.swift, you should modify getAdditionalInfo as well. For WIRE_ACCOUNT,  the additional info is different from BANK_ACCOUNT. The intermediaryBankAccountId should be used instead of bankAccountId for WIRE_ACCOUNT:

```
    func getAdditionalInfo(_ transferMethod: HyperwalletTransferMethod) -> String? {
        var additionlInfo: String?
        switch transferMethod.getField(fieldName: .type) as? String {
        case "BANK_ACCOUNT":
            additionlInfo = transferMethod.getField(fieldName: .bankAccountId) as? String
            additionlInfo = String(format: "%@%@",
                                   "transfer_method_list_item_description".localized(),
                                   additionlInfo?.suffix(startAt: 4) ?? "")
        case "BANK_CARD":
            additionlInfo = transferMethod.getField(fieldName: .cardNumber) as? String
            additionlInfo = String(format: "%@%@",
                                   "transfer_method_list_item_description".localized(),
                                   additionlInfo?.suffix(startAt: 4) ?? "")
        case "PAYPAL_ACCOUNT":
            additionlInfo = transferMethod.getField(fieldName: .email) as? String
        case "WIRE_ACCOUNT":
            additionlInfo = transferMethod.getField(fieldName: .intermediaryBankAccountId) as? String
            additionlInfo = String(format: "%@%@",
                                   "transfer_method_list_item_description".localized(),
                                   additionlInfo?.suffix(startAt: 4) ?? "")

        default:
            break
        }
        return additionlInfo
    }